Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py

```diff
@@ -2321,6 +2377,9 @@ def main():
         incidents_first_fetch = params.get("first_fetch", "3 days").strip()
         requests_count = int(params.get("max_fetch", 3))
 
+        exclude_combolist = params.get("exclude_combolist", False)
+        enable_probable_corporate_access = params.get("enable_probable_corporate_access", 0)
```

Comment:
There is no meaning for `0` here, because if this parameter is not checked then it will return `False`.
You can check in the `create_poll_generator` function if it is `False` then `enable_probable_corporate_access` will be equal to `0`.